File: Daily Challanges/2026-07-July/2026-07-22.py
def piggy_bank(coins):
    total = (
        coins.get("pennies", 0)
        + coins.get("nickels", 0) * 5
        + coins.get("dimes", 0) * 10
        + coins.get("quarters", 0) * 25
    )

    return f"${total / 100:.2f}"


print(piggy_bank({"pennies": 3, "nickels": 5, "dimes": 2, "quarters": 6}))
print(piggy_bank({"pennies": 1, "nickels": 1, "dimes": 1, "quarters": 1}))
print(piggy_bank({"nickels": 8, "dimes": 6, "quarters": 5}))
print(piggy_bank({}))
print(piggy_bank({"pennies": 146, "nickels": 11, "dimes": 0, "quarters": 19}))

File: Daily Challanges/2026-07-July/2026-07-22.js
function piggyBank(coins) {
  const total =
    (coins.pennies || 0) +
    (coins.nickels || 0) * 5 +
    (coins.dimes || 0) * 10 +
    (coins.quarters || 0) * 25;

  return `$${(total / 100).toFixed(2)}`;
}

console.log(piggyBank({ pennies: 3, nickels: 5, dimes: 2, quarters: 6 }));
console.log(piggyBank({ pennies: 1, nickels: 1, dimes: 1, quarters: 1 }));
console.log(piggyBank({ nickels: 8, dimes: 6, quarters: 5 }));
console.log(piggyBank({}));
console.log(piggyBank({ pennies: 146, nickels: 11, dimes: 0, quarters: 19 }));
